include/nbl/asset/filters/CBlitImageFilter.h

```diff
@@ -152,6 +152,84 @@ class CBlitImageFilter : public CImageFilter<CBlitImageFilter<Swizzle,Dither,Nor
 					return CScaledImageFilterKernel<Kernel>(fScale,kernel);
 				}
 
+				static void computePhaseSupportLUT(const CState* state)
+				{
+					const auto scaledKernelX = state->contructScaledKernel(state->kernelX);
+					const auto scaledKernelY = state->contructScaledKernel(state->kernelY);
+					const auto scaledKernelZ = state->contructScaledKernel(state->kernelZ);
+
+					const core::vectorSIMDu32 phaseCount = getPhaseCount(state->inExtentLayerCount, state->outExtentLayerCount, state->inImage->getCreationParameters().type);
+
+					const core::vectorSIMDu32 axisOffsets = getPhaseSupportLUTAxisOffsets(phaseCount, scaledKernelX, scaledKernelY, scaledKernelZ);
+
+					const core::vectorSIMDf fInExtent(state->inExtentLayerCount);
+					const core::vectorSIMDf fOutExtent(state->outExtentLayerCount);
+					const auto fScale = fInExtent.preciseDivision(fOutExtent);
+
+					// a dummy load functor
+					// does nothing but fills up the `windowSample` with 1s (identity) so we can preserve the value of kernel
+					// weights when eventually `windowSample` gets multiplied by them later in
+					// `CFloatingPointSeparableImageFilterKernelBase<CRTP>::sample_functor_t<PreFilter,PostFilter>::operator()`
+					// this exists only because `evaluateImpl` expects a pre filtering step.
+					auto dummyLoad = [](value_type* windowSample, const core::vectorSIMDf& unused0, const core::vectorSIMDi32& unused1, const IImageFilterKernel::UserData* unused2) -> void
+					{
+						for (auto h = 0; h < MaxChannels; h++)
+							windowSample[h] = value_type(1);
+					};
+
+					// There's no real reason that this should have `MaxChannels` number of elements except for the fact that
+					// `CFloatingPointSeparableImageFilterKernelBase<CRTP>::sample_functor_t<PreFilter,PostFilter>::operator()` expects that.
+					// So this, for now, just replicates the same value in all 4 channels.
+					value_type kernelWeight[MaxChannels];
+					// actually used to put values in the LUT
+					auto dummyEvaluate = [&kernelWeight](const value_type* windowSample, const core::vectorSIMDf& unused0, const core::vectorSIMDi32& unused1, const IImageFilterKernel::UserData* unused2) -> void
```

Comment:
instead of calling them `unused` you can just not give them names
https://stackoverflow.com/questions/2319663/why-does-c-code-missing-a-formal-argument-name-in-a-function-definition-compil

Reply:
Resolved in #343.